08-interop/08-interop.md

```diff
@@ -14,12 +14,98 @@ Task #02 - ICMPv6 echo between iotlab-m3 and Linux with 6LowPAN
 ICMPv6 echo request/reply exchange between an iotlab-m3 node running RIOT and
 a Raspberry Pi running Linux with 6LoWPAN support.
 
-Task #03 - ICMPv6 echo between iotlab-m3 and Contiki
-====================================================
+Task #03 - UDP exchange between iotlab-m3 and Contiki
+=====================================================
 ### Description
 
-ICMPv6 echo request/reply exchange between an iotlab-m3 node running RIOT and
-a Contiki node.
+UDP packet exchange between an iotlab-m3 node running RIOT and a Contiki node.
+
+### Testing procedure
+1. Get your hands on a [CC2538DK evaluation board][cc2538dk]
+1. Clone the Contiki repository to your machine
+   ```sh
+   git clone http://github.com/contiki-os/contiki.git && cd contiki/
+   ```
+2. Go to the `udp-ipv6-echo-server` for the CC2538DK:
+   ```sh
+   cd examples/cc2538dk/udp-ipv6-echo-server
+   ```
+3. Apply the following patch
+   ```sh
```

Comment:
It shouldn't be ```diff here? I think it's better formatted that way, unless you wanted to paste the actual comand to apply the patch.

Reply:
Yes, I wanted to paste the command to apply the patch. This way the tester just need to copy&paste ;-).

Reply:
However, I added the missing pipe below ;-D

Reply:
I used this to copy and paste...